src/appengine/handlers/cron/triage.py

```diff
@@ -307,6 +307,7 @@ def get(self):
       issue_tracker = issue_tracker_utils.get_issue_tracker_for_testcase(
           testcase)
       if not issue_tracker:
+        issue_filer.notify_issue_update(testcase, 'new')
```

Comment:
This isn't fully correct as _check_and_update_similar_bug check is done below. Should we just move the issue_tracker check below this ?

Reply:
_check_and_update_similar_bug needs issue_tracker to exist as well, so we can't move it.